daphne/src/vdaf/mod.rs

```diff
@@ -729,18 +732,27 @@ impl VdafConfig {
     /// * `state` is the helper's current state.
     ///
     /// * `agg_cont_req` is the aggregate request sent by the Leader.
-    pub(crate) fn handle_agg_cont_req(
+    pub(crate) fn handle_agg_job_cont_req(
         &self,
         state: DapHelperState,
-        agg_cont_req: &AggregateContinueReq,
+        agg_cont_req: &AggregationJobContinueReq,
         metrics: &DaphneMetrics,
-    ) -> Result<DapHelperTransition<AggregateResp>, DapAbort> {
+    ) -> Result<DapHelperTransition<AggregationJobResp>, DapAbort> {
+        if let Some(round) = agg_cont_req.round {
+            if round == 0 {
+                return Err(DapAbort::UnrecognizedMessage);
```

Comment:
Why not handle as RoundMismatch? Is this what the spec says to do?

Reply:
it is what the spec says!